commit db2292b01b799e926abfdbd6fafa1f27f0d0e457
Author: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Date:   Tue Dec 19 15:07:23 2023 +0100

    PM: clk: make pm_clk_add_notifier() take a const pointer
    
    The driver core wants to work with const struct bus_type, so there's no
    reason that pm_clk_add_notifier() should not also do the same thing,
    considering that it just passes the pointer off to the driver core which
    is expecting a const *.
    
    Cc: Rafael J. Wysocki <rafael@kernel.org>
    Link: https://lore.kernel.org/r/2023121922-triumph-exploit-f545@gregkh
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/base/power/clock_ops.c b/drivers/base/power/clock_ops.c
index 4110c19c08dc..e18ba676cdf6 100644
--- a/drivers/base/power/clock_ops.c
+++ b/drivers/base/power/clock_ops.c
@@ -793,7 +793,7 @@ static int pm_clk_notify(struct notifier_block *nb,
  * the remaining members of @clknb should be populated prior to calling this
  * routine.
  */
-void pm_clk_add_notifier(struct bus_type *bus,
+void pm_clk_add_notifier(const struct bus_type *bus,
 				 struct pm_clk_notifier_block *clknb)
 {
 	if (!bus || !clknb)